lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -235,11 +235,11 @@ <h3 class="course-title" id="course-title-${enrollment.course_id}">
                 % endif
             % endif
 
-            ## Right now, the gear dropdown for entitlements only contains the 'unenroll' link, so we should hide the
-            ## gear altogether if the user is unable to unenroll/refund their entitlement. Later, when we add more options
-            ## to the gear dropdown, we can remove this check.
-            % if entitlement and can_refund_entitlement:
-                <%include file='_dashboard_entitlement_actions.html' args='course_overview=course_overview,entitlement=entitlement,dashboard_index=dashboard_index, can_refund_entitlement=can_refund_entitlement'/>
+            ## We should only show the gear dropdown if the user is able to refund/unenroll from their entitlement
+            ## and/or if they have selected a course run and email_settings are enabled
+            ## as these are the only actions currently available
+            % if entitlement and (can_refund_entitlement or show_email_settings):
+                <%include file='_dashboard_entitlement_actions.html' args='course_overview=course_overview,entitlement=entitlement,dashboard_index=dashboard_index, can_refund_entitlement=can_refund_entitlement, entitlement_session=entitlement_session, show_email_settings=show_email_settings'/>
```

Comment:
`entitlement_session` does not seem to be used by `_dashboard_entitlement_actions.html`